modules/services/imapnotify-accounts.nix

```
@@ -0,0 +1,30 @@
+{ lib, ... }:
+
+with lib;
+
+{
+  options.imapnotify = {
+    enable = mkEnableOption "imapnotify";
+
+    onNotify = mkOption {
+      type = types.either types.str types.attrs;
```

The `attrs` should be avoided as much as possible since it sometimes results in unintuitive behavior, in particular when options are merged. Intead can have something like
```suggestion
      type = with types; either str (attrsOf str);
```